lib/src/arithmetic/parallel_prefix_operations.dart

```diff
@@ -161,54 +161,79 @@ class ParallelPrefixOrScan extends Module {
   Logic get out => output('out');
 
   /// OrScan constructor
-  ParallelPrefixOrScan(
-      Logic inp,
-      ParallelPrefix Function(List<Logic>, Logic Function(Logic, Logic))
-          ppGen) {
+  ParallelPrefixOrScan(Logic inp,
+      ParallelPrefix Function(List<Logic>, Logic Function(Logic, Logic)) ppGen)
```

Comment:
Should we add defaults on these too while we're at it?  Just to make them easier to use?

Reply:
Done